### DISCUSSION TITLE
Mapping changes in ocean heat content

### DISCUSSION THREAD

**Discussion by mnky9800n** [4DModeller/fdmr#305] [Ideas]
Here starts the conversation on ocean heat content.

**Comment by enoch26** [comment]
Barrier model tutorial https://eliaskrainski.github.io/INLAspacetime/articles/web/barrierExample.html

**Reply by enoch26** [reply]
https://raw.githubusercontent.com/hrue/r-inla/devel/rinla/vignettes/barrier-global/Tutorial_barrier_global.Rmd

**Comment by enoch26** [comment]
git@github.com:enoch26/fdmrOH.git

**Reply by enoch26** [reply]
https://github.com/enoch26/fdmrOH
